src/Laravel/Commands/IdeGenerateCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace SergiX44\Nutgram\Laravel\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\File;
+
+class IdeGenerateCommand extends Command
+{
+    protected $signature = 'nutgram:ide:generate';
+
+    protected $description = 'Generate IDE helper for Nutgram';
+
+    public function handle(): int
+    {
+        $this->info('Generating IDE helper...');
+
+        $helper = file_get_contents(__DIR__.'/../Stubs/Ide.stub');
```

Comment:
Wouldn't it be better to generate them via reflection?

Reply:
In my opinion it's too long to be done. It's just not worth it.